.github/workflows/publish-hybridse-test-result-from-fork.yml

```diff
@@ -8,66 +8,32 @@ on:
       - completed
 
 jobs:
-  test-results:
-    name: Test Results
-    runs-on: ubuntu-latest
+  test-results-linux:
     if: >
       github.event.workflow_run.conclusion != 'skipped' && (
          github.event.sender.login == 'dependabot[bot]' ||
          github.event.workflow_run.head_repository.full_name != github.repository
       )
+    name: Publish Linux UT Results
+    uses: team-317/OpenMLDB/.github/workflows/publish-test-result-from-fork-action.yml@reuse_workflow
+    with:
+      commit: ${{ github.event.workflow_run.head_sha }}
+      files: artifacts/linux-ut-result-*/**/*.xml
+      check_name: Linux Test Report
```

Comment:
`check_name` and `comment_title` should be `HybridSE Linux Test Report`

Reply:
Ok, thank you for your advice. I will revise it